packages/lesswrong/server/repos/TweetsRepo.ts

```diff
@@ -0,0 +1,69 @@
+import AbstractRepo from "./AbstractRepo";
+import Tweets from "../../lib/collections/tweets/collection";
+import { recordPerfMetrics } from "./perfMetricWrapper";
+import { getViewablePostsSelector } from "./helpers";
+
+class TweetsRepo extends AbstractRepo<"Tweets"> {
+  constructor() {
+    super(Tweets);
+  }
+
+  async getUntweetedPostsCrossingKarmaThreshold({threshold, since}: {threshold: number, since: Date}): Promise<string[]> {
+    const res = await this.getRawDb().any<{postId: string}>(`
+      -- TweetsRepo.getPostsCrossingKarmaThreshold
```

Comment:
Removing the extra query for `post_karma` should help a bit. I'm just using the autoformatting from my postgres client so I think this is about as good as it can get